### DISCUSSION TITLE
How to install Gramine on Ubuntu 23 ??

### DISCUSSION THREAD

**Discussion by mayank64ce** [gramineproject/gramine#2047] [General]
The package list at https://packages.gramineproject.io/ does not list any package for Ubuntu 24 `mantic` distribution.

How can I install on Ubuntu 23 ?

**Comment by kailun-qin** [comment]
@mayank64ce: Reading https://github.com/gramineproject/gramine/issues/2048, it looks like you've already solved this issue yourself. Closing.
